### PR TITLE
#54: Sign In to CMS

### PR DESCRIPTION
**Pull request by exymore**
- [x] Next.JS
- [x] Redux, Redux-Saga
- [x] Atomic Design Structure
- [x] Log In Screen

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,6 @@ typings/
 
 # next.js build output
 .next
+
+.idea
+.vscode
```
Comment (underscorekadji) [inline .gitignore]: This folder may contain useful settings for working with visual studio code.

File: src/services/authService.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  const res = await fetch(LOGIN_ENDPOINT, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(req.user),
+  })
+
+  const json = await res.json()
+  console.log(json)
```
Comment (underscorekadji) [inline src/services/authService.js]: console.log
add linter warning for console.log

File: src/utils/colorVars.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export const colors = {
```
Comment (underscorekadji) [inline src/utils/colorVars.js]: My opinion that this enum must be somewhere in theme folder.

File: src/services/authService.js (hunk 2 above)
Comment (exymore) [reply src/services/authService.js]: already added

Comment (underscorekadji) [reply src/services/authService.js]: so remove console.log(json) :)

File: pages/_app.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import App, { Container } from 'next/app'
+import React from 'react'
+import { Provider } from 'react-redux'
+import withReduxStore from '../lib/with-redux-store'
+
+class MyApp extends App {
```
Comment (underscorekadji) [inline pages/_app.js]: I think MyApp isn't good name in this case :)

**Pull request merged**